splunklib/searchcommands/internals.py

```diff
@@ -54,7 +51,7 @@ def set_binary_mode(fh):
     if sys.version_info >= (3, 0) and hasattr(fh, 'buffer'):
         return fh.buffer
     # check for python3
-    elif sys.version_info >= (3, 0):
+    if sys.version_info >= (3, 0):
```

Comment:
Since we are removing support for Python 2 entirely in this MR, this method needs more simplification to remove the checks for sys.version_info - these are no longer necessary.

Reply:
yes, simplified the method and removed the unnecessary code